ci/jenkins/jobs/projects.yaml

```diff
@@ -377,6 +377,197 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{test_name}-for-pull-request':
+          test_name: nephe-test-e2e-eks-with-agent
+          node: 'nephe'
+          description: 'This is the {test_name}.'
+          branches:
+            - ${{sha1}}
+          builders:
+            - deploy-vm-with-agent-and-run-eks-tests-for-nephe:
+                conformance_type: 'networkpolicy'
```

Comment:
change line 387 and 388 to
`- deploy-vm-with-agent-and-run-eks-tests-for-nephe`

Reply:
so does line 484 and 485